src/shared/ui/date-simple/date-simple.tsx

```
@@ -1,61 +1,66 @@
+      )}
+
+      {labelSeparator && endDate && <span>{labelSeparator}</span>}
+      {endDate && (
+        <time dateTime={endDateAttr} data-testid="date-time-end">
+          {endDate}
+        </time>
+      )}
+      {children}
+    </p>
+  );
+};
```

ESLint

[error] Newline required at end of file but not found.

(@stylistic/eol-last)